src/libawkward/io/json.cpp

```diff
@@ -125,7 +137,17 @@ namespace awkward {
 
   void
   ToJsonString::real(double x) {
-    impl_->real(x);
+    // FIXME: what about negative 'NaN'?
```

Comment:
There is no negative NaN, so this is right.

Reply:
done